modules/apps/journal/journal-web/src/main/resources/META-INF/resources/js/UndoRedo.js

```diff
@@ -109,82 +256,148 @@ export default function UndoRedo({
 				`${portletNamespace}defaultLanguageId`
 			);
 
-			const descriptionInputComponent = Liferay.componentReady(
-				`${portletNamespace}${META_FIELD_NAMES.description}`
+			const selectedLanguageIdInput = document.getElementById(
+				`${portletNamespace}languageId`
 			);
 
-			const titleInputComponent = Liferay.componentReady(
-				`${portletNamespace}${META_FIELD_NAMES.title}`
-			);
+			const newHistory = {
+				defaultLanguageId: defaultLanguageIdInput.value,
+				descriptionInputComponent:
```

Comment:
Maybe we could rename this to `descriptionInputValue`, so we don't confuse with the component? Same with friendlyURLInputComponent->friendlyURLInputValue and titleInputComponent-> titleInputValue